app/models/financialDetails/YourSelfAssessmentChargesViewModel.scala

```diff
@@ -29,6 +29,7 @@ case class YourSelfAssessmentChargesViewModel(hasOverdueOrAccruingInterestCharge
                                               dunningLock: Boolean,
                                               reviewAndReconcileEnabled: Boolean,
                                               creditAndRefundEnabled: Boolean,
+                                              earliestTaxYearAndAmountByDueDate: (TaxYear, BigDecimal),
```

Comment:
if we can avoid using tuples in this fashion as this would generate "not readable code";
feel free to create a new case class if you need to in order to improve code readability